pkg/restic/restorer.go

```diff
@@ -136,13 +145,13 @@ ForEachVolume:
 	}
 
 	r.resultsLock.Lock()
-	delete(r.results, resultsKey(pod.Namespace, pod.Name))
+	delete(r.results, resultsKey(data.Pod.Namespace, data.Pod.Name))
 	r.resultsLock.Unlock()
 
 	return errs
 }
 
-func newPodVolumeRestore(restore *velerov1api.Restore, pod *corev1api.Pod, volume, snapshot, backupLocation, repoIdentifier string) *velerov1api.PodVolumeRestore {
+func newPodVolumeRestore(restore *velerov1api.Restore, pod *corev1api.Pod, backupLocation, volume, snapshot, repoIdentifier string) *velerov1api.PodVolumeRestore {
```

Comment:
Moved the order here only to make it so all elements of the `RestoreData` struct were the first (3) inputs.